contracts/bond/ERC20SingleCollateralBond.sol

```diff
@@ -31,6 +31,13 @@ contract ERC20SingleCollateralBond is
     Redeemable,
     Version
 {
+    struct SlashData {
```

Comment:
just `Slash`?

Reply:
`Slash` 👍

Reply:
The compiler doesnt like having an event called `Slash` and a struct called `Slash` it appears

Reply:
Oh no!

Maybe name the event `SlashDeposits` as we are slashing the deposits? 🤷